dxr/config.py

```diff
@@ -19,6 +20,11 @@
 from dxr.plugins import all_plugins_but_core, core_plugin
 from dxr.utils import cd, if_raises
 
+#setup default elasticsearch host to use if not specifice in any config file
+DXR_DEFAULT_ELASTICSEARCH_HOST = getenv('DXR_DEFAULT_ELASTICSEARCH_HOST', '127.0.0.1')
```

Comment:
If we don't end up using this in serve.py, then we can collapse it into the next line.